documentation/src/main/asciidoc/user_guide/clustering.adoc

```diff
@@ -1013,6 +1014,14 @@ Currently Infinispan provides the following implementations of EntryMergePolicy:
 
 | MergePolicies.REMOVE_ALL
 | Always remove a key from the cache when a conflict is detected.
+
+| MergePolicies.NONE
+| No attempt is made to resolve conflicts. Instead, entries hosted on the minority partition are removed and the nodes in this partition do not hold any data until the rebalance starts. Note, this behaviour is equivalent to prior Infinispan versions which did not support conflict resolution.
+Note, in this case all changes made to one (the part with less nodes) side of the split are lost. But the state after rebalancing will be consistent.
+
+| Full qualified class name
+|Instead of setting a included MergePolicy a custom merge policy implementation can be added by set the full qualified class name of the custom implementation for the org.infinispan.conflict.EntryMergePolicy<K,V>.
+TODO: Note that implementation need to know what kind of Objects are provided, this is different in embedded or client/server mode. See XXXXXXXXXXX
```

Comment:
I don't think this section belongs in this table, as this table details the EntryMergePolicy implementations provided by Infinispan. I think a separate section detailing custom implementations would be better.

Reply:
Actually, this is already shown in the "Configuring partition handling" section.  We could just explicity state in the existing section that you must provide the FQN of the implementation.